### DISCUSSION TITLE
Legit the undeniable best config hand crafted by the best beta member helpmop and #1 femboy gl dont care for 2b2t

### DISCUSSION THREAD

**Discussion by HelpMopDog** [DarkiBoi/konas-issues#354] [Configs]
![2021-05-01_15 37 52](https://user-images.githubusercontent.com/68680918/116796725-bc3d9900-aa93-11eb-82d9-20a8413e232a.png)
![2021-05-01_15 37 57](https://user-images.githubusercontent.com/68680918/116796730-c1024d00-aa93-11eb-8534-63b1c6d55922.png)
![2021-05-01_15 38 07](https://user-images.githubusercontent.com/68680918/116796733-c65f9780-aa93-11eb-9859-97a7f878a626.png)
![2021-05-01_15 38 10](https://user-images.githubusercontent.com/68680918/116796736-ca8bb500-aa93-11eb-9dd2-9d76fc315e52.png)
![2021-05-01_15 38 16](https://user-images.githubusercontent.com/68680918/116796739-ce1f3c00-aa93-11eb-9580-ed6d720412da.png)
![2021-05-01_15 38 23](https://user-images.githubusercontent.com/68680918/116796740-d37c8680-aa93-11eb-845f-4270725c2992.png)
![2021-05-01_15 38 27](https://user-images.githubusercontent.com/68680918/116796743-d7100d80-aa93-11eb-9da5-172bb845ba6d.png)
![2021-05-01_15 38 30](https://user-images.githubusercontent.com/68680918/116796745-daa39480-aa93-11eb-8abc-61b8862b5ced.png)

**Comment by HelpMopDog** [comment]
If I repeated any screenshots idc and if your try and argue that these  arent the best settings you will be promply banned from konas discord and client for undermining.

**Reply by NotJohnmcswag** [reply]
bad config

**Reply by supakeks** [reply]
Shit config

**Reply by HelpMopDog** [reply]
Kys fat jews

**Reply by FIshyZero** [reply]
Shit config
